docker/config/devnet/tendermint/genesis.json

```diff
@@ -1,5 +1,5 @@
 {
-    "app_hash": "9B3263E4412E90902E7ED634399062A8DEDF45E89C87919E1474FD23EF77BB2F",
+    "app_hash": "D3218ECBF9D1F90656C624246ECDFB30CE01386F5BD5C6AC423766FEFC0DEB62",
```

Comment:
is this needed?
why not to update https://github.com/crypto-com/chain/blob/master/docker/config/devnet/dev-conf.json with it?